static/gsApp/registerHooks.tsx

```diff
@@ -121,7 +122,15 @@ const GETSENTRY_HOOKS: Partial<Hooks> = {
   'sidebar:organization-dropdown-menu': hookSidebarDropdownMenu,
   'sidebar:help-menu': hookSidebarHelpMenu,
   'sidebar:item-label': () => LabelWithPowerIcon,
-  'sidebar:bottom-items': props => <TryBusinessSidebarItem {...props} />,
+  'sidebar:bottom-items': props => (
+    <Fragment>
+      <TryBusinessSidebarItem key="try-business-sidebar-item" {...props} />
```

Comment:
Since this requires props it might be better to create separate hooks for these, like `sidebar:try-business` and `sidebar:quota-exceeded`.